executor/point_get.go

```diff
@@ -385,6 +385,10 @@ func (e *PointGetExecutor) get(ctx context.Context, key kv.Key) ([]byte, error)
 			return val, nil
 		}
 	}
+	// Global temporary table is always empty, so no need to send the request.
+	if e.tblInfo.TempTableType == model.TempTableGlobal {
+		return nil, nil
+	}
```

Comment:
How about move this part before the PointGetCache block? Do temporary tables support point get cache?

Reply:
We should not support `PointGetCache` for temporary table.
That is meaningless: temporary table is in-memory already, so why should we use a another cache to complicate the problem without improving performance?